api/phpunit.nodocker.xml

```diff
@@ -13,6 +13,7 @@
         <server name="SHELL_VERBOSITY" value="-1" />
         <server name="SYMFONY_PHPUNIT_REMOVE" value="" />
         <server name="SYMFONY_PHPUNIT_VERSION" value="9.5.10" />
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="quiet[]=indirect&amp;quiet[]=other"/>
```

Comment:
What are "other" deprecations?

Reply:
The documentation is a bit scarce on what it is exactly. Before this PR, `self` and `direct` was always concerning our own code and and `indirect` and `other` was always concerning library code. At the moment, we only have 1 direct remaining and 25 indirects:

```
Remaining direct deprecation notices (1)

  1x: The "ApiPlatform\Core\Bridge\Doctrine\Orm\Util\EagerLoadingTrait" trait is considered internal. It may change without further notice. You should not use it from "App\Doctrine\EagerLoadingExtension".
    1x in CreateActivityTest::setUp from App\Tests\Api\Activities

Remaining indirect deprecation notices (25)
```

Reply:
Then would it make sense to not ignore "other" for now, and start ignoring them when we encounter them and can confirm they don't affect us?

Reply:
Sounds good for me.
Re-enabled with https://github.com/ecamp/ecamp3/pull/2370/commits/42a26a030a152d633207081ebab2ea2ef4f07342